### ISSUE TITLE
Need sorting options within subreddits and comments

### ISSUE THREAD

**Issue by ReddhubApp** [ferasm/reddhub#264]
I don't see sorting options anywhere, it would be great to have that.

**Comment by ohnokiadidnt** [comment]
Agreed.  I was going to submit this as feedback/suggestion, but didn't want to duplicate.  Need the ability to sort by oldest, newest, most viewed, hot, controversial, etc.
